features/facebookImport/src/model/importers/utils/execution-result.js

```diff
@@ -0,0 +1,28 @@
+import { createSuccessStatus } from "../../analyses/utils/analysis-status";
```

Comment:
Don't really get why we have the error status and the success status in separate places. I guess it's because we're decoupling analysis from importing, but in that case importing can also be successful, so what we're talking is about creating different statuse (stati?) here.